src/components/mgt-people-picker/mgt-people-picker.scss

```
@@ -15,9 +15,23 @@ $avatar-size: var(--avatar-size, 48px);
 $initials-color: var(--initials-color, white);
 $initials-background-color: var(--initials-background-color, #{$ms-color-magenta});
 $accent-color: var(--accent-color, #0078d4);
-$separator-margin: var(--separator-margin, 0px 0px 0px 0px);
 
-$people-list-background-color: var(--people-list-background-color, white);
+$input-border: var(--input-border, 2px solid #b3b0ad);
+$input-border-bottom: var(--input-border-bottom, $input-border);
+$input-border-right: var(--input-border-right, $input-border);
+$input-border-left: var(--input-border-left, $input-border);
+$input-border-top: var(--input-border-top, $input-border);
+
+$input-hover-color: var(--input-hover-color, #323130);
+$input-focus-color: var(--input-focus-color, #0078d4);
+$input-background-color: var(--input-background-color, white);
```

Removing `--people-list-background-color` is a breaking change, but we can avoid it by chaining the vars

```suggestion
$input-background-color: var(--input-background-color, var(--people-list-background-color, white));
```

---

yup works well: 
3370fa4